SPECS/python-platformdirs/python-platformdirs.spec

```diff
@@ -26,6 +25,12 @@ BuildRequires:  python3-pathspec
 BuildRequires:  python3-pluggy
 BuildRequires:  python3-tomli
 BuildRequires:  python3-trove-classifiers
+%if 0%{?with_check}
+BuildRequires:  python3-pytest
+BuildRequires:  python3-pytest-mock
+BuildRequires:  python3-appdirs
+BuildRequires:  python3-iniconfig
```

Comment:
Looking at the build, I'm guessing we don't have `python3-iniconfig` for the tests. In that case, while not ideal, it's OK to `pip3 install iniconfig==<version>` in the `%check` section to satisfy the missing dependency. The `==<version>` part helps to keep the tests stable and break only due to changes in AZL Python packages.

Reply:
Hi @pawel, I updated the `iniconfig` installation step under the check section. Thanks for your input.

Reply:
Thank you, @0xba1a!